tests/samplers_tests/tpe_tests/test_multi_objective_sampler.py

```diff
@@ -311,7 +311,8 @@ def test_multi_objective_sample_independent_ignored_states() -> None:
 )
 @pytest.mark.parametrize("objective_value", [-5.0, 5.0, 0.0, -float("inf"), float("inf")])
 @pytest.mark.parametrize("multivariate", [True, False])
-@pytest.mark.parametrize("constant_liar", [True, False])
+# TODO(not522): Test constant_liar=True
```

Comment:
Why does not this test `constant_liar=True`?

Reply:
I fixed the test as I realized we should ignore the `constant_liar` option as documented.

Reply:
I see. Thanks for the explanation.